### PR TITLE
Quilt logging updates

### PR DESCRIPTION
**Pull request by mmussomele**
I was working on these changes as part of the scale tester, but they aren't relevant to it anymore, so it can't hurt to merge them now.

### CODE REVIEW

File: quilt.go [hunk 1]
```diff
@@ -32,8 +32,8 @@ func main() {
 	flag.Usage = func() {
 		fmt.Println("Usage: quilt " +
 			"[-log-level=<level> | -l=<level>] [-H=<listen_address>] " +
-			"[inspect <stitch> | run <stitch> | minion | " +
-			"stop <namespace> | get <import_path> | " +
+			"[out=<output_file>] [inspect <stitch> | run <stitch> | " +
```
Comment (ejj) [inline quilt.go]: I don't like `out`.  Maybe `--log-file` instead?

**Comment by ZhongtianWang** [comment]
same comment as ejj. Other than that, everything looks good! Oh and you need to rebase now.

**Comment by mmussomele** [comment]
fixed and pushed

**Comment by ejj** [comment]
Merged

**Pull request closed without merging**